openapi.yaml

```diff
@@ -2032,82 +2228,29 @@ paths:
   /process_graphs:
     get:
       summary: List all user-defined processes
-      operationId: list-custom-processes
+      operationId: list-custom-processes-legacy
+      deprecated: true
       description: |-
-        Lists all user-defined processes (process graphs) of the
+        Redirects to all user-defined processes of the
```

Comment:
Also add some kind of deprecation notice to the description of these endpoints?

Reply:
At least it should probably describe what to use/do instead. But I'm still not sure whether to use this redirect behavior (breaking) or leave /process_graphs as it is (basically as an alias, but non-breaking).

Reply:
how do you mean that the redirect is breaking? That (some) clients don't properly handle a redirect on POST/PUT/PATCH/DELETE?

Reply:
Yes, I assume that not all clients would handle it properly. But if we test JS, R and Python and all work flawlessly, it's probably okay to call it non-breaking ;-)